std/src/main/scala/algebra/std/map.scala

```diff
@@ -26,33 +50,80 @@ class MapEq[K, V](implicit ev: Eq[V], a: AdditiveMonoid[V]) extends Eq[Map[K, V]
     check(x, y) && check(y, x)
```

Comment:
We have be burned REPEATEDLY by this question. Writing tests on Map algebras is a massive pain if you don't require normalization in the operations. But since inputs are not necessarily normalized, you have pains (so you can normalize them, but then in use who knows if they are normalized, etc...)

I think it would be good to have two variants: mapVectorEq and and mapEq. For mapVectorEq we subtract left from right and then the result has all values zero. This is the approach we finally had in tests (but Algebird has no Eq typeclass).